bin/webpack-entries.js

```diff
@@ -45,6 +45,9 @@ const blocks = {
 	'stock-filter': {},
 	'active-filters': {},
 	cart: {},
+	'cart-order-summary-taxes-block': {
+		customDir: 'cart/inner-blocks/cart-order-summary-taxes',
+	},
```

Comment:
I question the need for this one and maybe we can just rely on cart.js file :/

Reply:
When I load the page (the cart edit mode), I get this error:

<img width="616" alt="image" src="https://user-images.githubusercontent.com/14235870/181003177-bb937323-a49a-4ee1-bed5-8c2fdcc2914c.png">

And after adding the code above, it stopped showing this console error. I think now that the `taxes'` inner block is registered server side, it's expecting to have its `JS` file. But I'll look into that and see If we can choose not to include a JS file in this case.

Reply:
Fixed here: [c5670f6](https://github.com/woocommerce/woocommerce-blocks/pull/6737/commits/c5670f6be17c2de2552028dc959bbcb8f4e57055) ✅